js/runner.go

```diff
@@ -158,6 +159,21 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		}
 	}
 
+	pool, _ := x509.SystemCertPool()
+	if pool == nil {
+		// SystemCertPool is not available on Windows.
+		// x509.SystemCertPool will always fail in that environment.
+		// The error can be ignored, instead, we initialize a new pool.
+		pool = x509.NewCertPool()
```

Comment:
That will probably need testing ... :sob:

Reply:
yeah, I've done it in the past, but I'll have to check how tests bootstrap data in this project to do that.